addon/src/lib/middleware/Hub.js

```diff
@@ -14,8 +14,8 @@ export default class Hub {
   dispatch: Dispatch;
   ports: Set<EventEmitter>;
   constructor() {
-    this.dispatch = () =>
-      console.error('Hub cannot use dispatch() before middleware()');
+    this.dispatch = () => // eslint-disable-next-line no-console
```

Comment:
Had to read this a couple times... looks like an arrow function to nowhere, but I assume it in fact references the `console.error` on the following line (which is now unindented below. Not sure if we want to throw some free `{`..`}` around the arrow function to make it clearer (or not, whatever).

But it passes `npm run lint` and still seems to lint after running `npm run format`.